components/Panel/StakeUnstakePanel/Stake.tsx

```diff
@@ -62,6 +64,10 @@ export function Stake({
     );
   }
 
+  function isInputDisabled() {
```

Comment:
why not just use the value directly instead of using a function?

Reply:
no reason really, just noticed there were other similar functions so following the pattern, also its better for clarity